extensions/build.gradle

```diff
@@ -0,0 +1,10 @@
+subprojects { subproject ->
+	plugins.withId("java") {
+		task deploy(type: Copy, dependsOn: jar) {
+			from jar
+			into new File(System.getProperty("user.home") + "/.blade/extensions")
```

Comment:
This messes up tests :( and can cause failures on CI or in docker, so we need to discuss this one.

Reply:
why would it fail CI and in docker.  just don't use the taks?

Reply:
Oh i see, this is a deploy task, yeah that is safe.  I was thinking this would happen before any of our unit tests which would be aproblem

Reply:
Good point, I was thinking this would interfere with tests, but then I realize all of our tests have isolated extensions directories (never use real user home) to prevent that problem, so yeah we should be good.